bootloader/src/pyi_win32_utils.c

```diff
@@ -435,6 +435,55 @@ pyi_win32_utf8_to_mbs(char * dst, const char * src, size_t max)
     }
 }
 
+
+/* Retrieve the SID of the current user.
+ *  Returns SID string on success, NULL on failure.
+ */
+static char *
+pyi_win32_get_user_sid()
```

Comment:
Mark es "private" - python style
```suggestion
_pyi_win32_get_user_sid()
```

Reply:
Done.